Modules/Challenges/test/Challenges.API.Tests.Performance.EndToEnd/src/utils.ts

```diff
@@ -0,0 +1,58 @@
+import http from "k6/http";
+import exec from "k6/execution";
+
+export interface Configuration {
+  Host: string;
+  Client_Secret: string;
+  User: string;
+  Password: string;
+  Size: string;
+}
+
+export interface Size {
+  vus: number;
+  iterations: number;
+}
+
+export function getAuthenticationHeader(configuration: Configuration): string {
+  const bodyConnectToken = {
+    client_id: "test",
+    client_secret: configuration.Client_Secret,
+    username: configuration.User,
+    password: configuration.Password,
+    grant_type: "password",
+  };
+
+  const authToken = http
+    .post(`${configuration.Host}/connect/token`, bodyConnectToken, {
+      headers: {
+        "Content-Type": "application/x-www-form-urlencoded",
+      },
+    })
+    .json("access_token")!
+    .toString();
+
+  return `Bearer ${authToken}`;
+}
+
+export function getConfiguration(): Configuration {
+  assertEnvVarExists("HOST");
```

Comment:
We should probably validate that the HOST doesn't have a trailing `/`. Because otherwise our concatenated baseUrl in the tests would look like `localhost:5000//api/v1`.

Just do it for the files performance tests in the same PR. Such minor things don't need to be separated.